pkg/pgmodel/end_to_end_tests/delete_test.go

```diff
@@ -0,0 +1,481 @@
+package end_to_end_tests
```

Comment:
we should add a test that sends a request to the delete endpoint. Currently the endpoint itself isn't tested

Reply:
Done.